### DISCUSSION TITLE
@机器人时，两个插件同时被触发。如何设置使得只有一个插件被触发？

### DISCUSSION THREAD

**Discussion by Loss-of-time** [nonebot/discussions#289] [问答] [answered]
被触发的两个插件
都是@机器人+文本消息的形式
nonebot_plugin_chatgpt 聊天机器人
nonebot_plugin_tts_gal VITS语音生成

**Comment by yanyongyu** [comment, marked as answer]
如果插件没有提供配置优先级的方法，那没有办法。你可以去插件仓库提issue。
